translator/config/sampleSchema/validContainerInsightsJmx.json

```diff
@@ -0,0 +1,16 @@
+{
+  "agent": {
+    "region": "us-west-2"
+  },
+  "logs": {
+    "metrics_collected": {
+      "emf": {
+      },
+      "kubernetes": {
+        "cluster_name": "jvmCluster",
+        "namespace":"siprmp-jmx"
```

Comment:
nit - this should be updated to a non identifying namespace for testing

Reply:
removed namespace since it isn't even used (underneath kubernetes)